plugins/azure/loadbalancer/lbPublicIp.spec.js

```diff
@@ -0,0 +1,152 @@
+var expect = require('chai').expect;
+var lbPublicIp = require('./lbPublicIp');
+
+const loadBalancers = [
+    {
+        "name": "test",
+        "id": "/subscriptions/dce7d0ad-ebf6-437f-a3b0-28fc0d22117e/resourceGroups/testresource/providers/Microsoft.Network/loadBalancers/test",
```

Comment:
```suggestion
        "id": "/subscriptions/dce7d0ad-ebf6-437f-a3b0-28fc0d22117e/resourceGroups/testresource/providers/Microsoft.Network/loadBalancers/test",
```
are these dummy values?